packages/visx-annotation/src/components/Label.tsx

```diff
@@ -67,7 +69,78 @@ function getCompletePadding(padding: LabelProps['backgroundPadding']) {
   return { ...DEFAULT_PADDING, ...padding };
 }
 
-export default function Label({
+export default function Label({ ...props }: LabelProps) {
+  if (props.children) {
+    return <ForeignObjectLabel {...props} />;
+  }
+  return <SvgLabel {...props} />;
+}
+
+function ForeignObjectLabel({
```

Comment:
wondering if we could pull this out into a separate file since this one is getting pretty large.

Reply:
Couple more thoughts:
- What do you think about calling it `HtmlLabel` instead? that might be more understandable by folks who don't know what `foreignObject` means. And `foreignObject` will still be documented via props.
- I think especially because this label type is so simplified / supports a subset of props, users might want to import and use it directly, so a separate file would be good. 
- For documentation purposes, it would probably also be ideal to explicitly define the subset of props it supports with `Pick<LabelProps, ...>` because otherwise users may wonder why some don't have an effect. 
- If separated into another file, for this component to show up in the docs we'll need to add it [here](https://github.com/airbnb/visx/blob/master/packages/visx-demo/src/pages/docs/annotation.tsx#L19) and export it from the package [`index.ts`](https://github.com/airbnb/visx/blob/master/packages/visx-annotation/src/index.ts#L2)